sourcecode/apis/contentauthor/app/Http/Controllers/ContentAssetController.php

```diff
@@ -4,72 +4,21 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
-use Illuminate\Support\Facades\Cache;
 use Illuminate\Support\Facades\Storage;
-use Illuminate\Http\Response;
-use League\Flysystem\FileNotFoundException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 class ContentAssetController
```

Comment:
Any chance we could get rid of this entirely and rely on CDN prefix + file name instead?

Reply:
No because we need to be able to run edlib without a CDN.

Reply:
I meant public path prefix, doesn't matter if it's backed by a CDN or not.

Reply:
We actually have 3 different storing/retrieving methods now. 

1. Locally: We are storing to the public folder under /h5pstorage and the nginx proxy just serves the assets
2. In prod without CDN: We are storing to s3 and using the controller to serve those assets
3. In prod with CDN: We are storing to s3 and using the CDN to view the assets.

Ideally we should only have 2 different setup, with and without CDN. When developing locally we could for example just change the location of the files to a folder instead of s3. This would simplify our setup a lot.

All of this is out of scope for this issue but we should definitely do something about it.